### PR TITLE
saml login

### PR DESCRIPTION
**Pull request by iamtarun1993**

### CODE REVIEW

File: capacitor.config.json [hunk 1]
```diff
@@ -9,5 +9,8 @@
       "launchShowDuration": 0
     }
   },
```
Comment (iamtarun1993) [inline capacitor.config.json:11]: will revert this file changes later

File: src/app/auth/sign-in/sign-in.page.ts [hunk 2]
```diff
@@ -27,9 +28,53 @@ export class SignInPage implements OnInit {
...
+      const samlNewRefreshToken$ = this.authService.newRefreshToken(data.refresh_token);
+
+      samlNewRefreshToken$.subscribe(() => {
+        this.router.navigate(['/', 'auth', 'switch-org']);
+      });
+    }
+  };
+
+  handleSamlSignIn (res) {
+    var url = res.idp_url + '&RelayState=MOBILE';
```
Comment (Julias0) [inline src/app/auth/sign-in/sign-in.page.ts:54]: use let & const instead if var

Comment (Julias0) [reply src/app/auth/sign-in/sign-in.page.ts:54]: modify tslint file and use ng lint to have these come up in terminal

**Review by Julias0 — APPROVED** [review]
PR lgtm - minor comments

**Pull request closed without merging**